views/homework/assignment.hbs

```diff
@@ -1,150 +1,153 @@
-                                {{> 'homework/submissions'}}
-                            {{/if}}
-                        {{/if}}
-                    {{/if}}
-                </div>
-            </div>
-        </section>
-    {{/content}}
-{{/extend}}
+{{#content "page"}}
+<section class="section-homework">
+    <div class="dates text-muted">
+        <i
+            class="fa fa-{{#if submissions}}check done{{else if warning}}exclamation-triangle danger{{else}}{{#unless submittable}}ban closed{{/unless}}{{/if}}"></i>
+        {{availableDateF}} ({{availableTimeF}}) |
+        {{#if dueDate}}
+        bis: {{dueDateF}} ({{dueTimeF}})
+        {{else}}
+        {{$t "homework.text.noSubmissionDateSet"}}
+        {{/if}}
+    </div>
+    {{#if courseId._id}}
+    <div class="col-sm-12">
+        <div class="pull-right">
+            <a href="/files/courses/{{courseId._id}}" class="btn btn-add btn-secondary">
+                <i class="fa fa-folder-open"></i>
+                {{$t "homework.button.toTheCourseFiles" }}
+            </a>
+            <a class="btn btn-secondary" href="/courses/{{courseId._id}}/?activeTab=homeworks"
+                alt="Zum Kurs der Aufgabe">
+                <i class="fa fa-graduation-cap"></i>
+                {{$t "homework.button.toClass" }}
+            </a>
+            {{#if lessonId}}
+            <a class="btn btn-secondary" href="/courses/{{courseId._id}}/topics/{{lessonId}}"
+                alt="Zum Thema der Aufgabe">{{$t "homework.label.onTheSubject" }}</a>
+            {{/if}}
+        </div>
+    </div>
+    {{/if}}
+    <div class="tab-view maintabs">
+        <nav class="nav tab-links">
+            <a id="extended-tab-link" class="tab-link active" href="#activetabid=extended">
+                {{$t "homework.label.details" }}
+            </a>
+            {{#if private}}
+            <a id="submission-tab-link" class="tab-link" href="#activetabid=submission">
+                {{$t "homework.label.submission" }}
+            </a>
+            {{else}}
+            {{#if (userHasRole "teacher")}}
+            <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions">
+                {{$t "homework.label.submissions" }}
+            </a>
+            {{else}}
+            <a id="submission-tab-link" class="tab-link" href="#activetabid=submission">
+                {{$t "homework.label.submission" }}
+            </a>
+            <a id="feedback-tab-link" class="tab-link" href="#activetabid=feedback">
+                {{$t "homework.label.rating_comment" }}
+            </a>
+            {{#if ../publicSubmissions}}
+            <a id="submissions-tab-link" class="tab-link" href="#activetabid=submissions">
+                {{$t "homework.label.submissions" }}
+            </a>
+            {{/if}}
+            {{/if}}
+            {{/if}}
+        </nav>
+        <div class="tabs">
+            {{> 'homework/extended'}}
+            {{#if private}}
```

Comment:
@dominikbruechner how we write if else in handelbars? All in line, or with tab for body?